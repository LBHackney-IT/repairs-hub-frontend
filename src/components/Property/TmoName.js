import PropTypes from 'prop-types'

const TmoName = ({ tmoName }) => (
  <>{tmoName && <li className="bg-orange">TMO: {tmoName}</li>}</>
)

TmoName.propTypes = {
  tmoName: PropTypes.string,
}

export default TmoName
