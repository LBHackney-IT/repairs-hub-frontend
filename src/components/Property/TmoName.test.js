import { render } from '@testing-library/react'
import TmoName from './TmoName'

describe('TmoName component', () => {
  const props = {
    tmoName: 'Test address TMO',
  }

  it('should render properly', async () => {
    const { asFragment } = render(<TmoName tmoName={props.tmoName} />)
    expect(asFragment()).toMatchSnapshot()
  })
})
