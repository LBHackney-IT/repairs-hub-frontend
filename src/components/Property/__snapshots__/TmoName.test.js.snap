// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TmoName component should render properly 1`] = `
<DocumentFragment>
  <li
    class="bg-orange"
  >
    TMO: Test address TMO
  </li>
</DocumentFragment>
`;
